### ISSUE TITLE
Currently impossible to listen on a port

### ISSUE THREAD

**Issue by floitsch** [toitware/public#7]
Currently the following doesn't work.
```
import net

main :
  network_interface := net.open
  print "network_interface -- ok"
  serverSocket := network_interface.tcp_listen 8080
  serverSocket.listen "localhost"
  print "wait..."
  localAddr := serverSocket.local_address
  socket := serverSocket.accept
  print "Accept was done..."
```

The following code has been found to be the culprit:
```
 if (!address->slow_equals("localhost") && !address->slow_equals("127.0.0.1")) {
      // We currently only implement binding to localhost or INADDR_ANY.
      UNIMPLEMENTED_PRIMITIVE;
    }
```
We need to allow more addresses here.

**Comment by kasperl** [comment]
We have made progress on this in SDK v1.1.0, but we have found an additional issue.

**Comment by kasperl** [comment]
We have just shipped SDK v1.1.1 which solves the last known issue with port listening.

**Comment by kasperl** [comment]
The solution provided in SDK v1.1.1 has been verified and listening now works.
